apps/meteor/server/lib/pushConfig.ts

```diff
@@ -9,6 +10,48 @@ import { Push } from '../../app/push/server';
 import { settings } from '../../app/settings/server';
 import { i18n } from './i18n';
 
+export const executePushTest = async (userId: IUser['_id'], username: IUser['username']): Promise<number> => {
+	const query = {
+		$and: [
+			{
+				userId,
+			},
+			{
+				$or: [
+					{
+						'token.apn': {
+							$exists: true,
+						},
+					},
+					{
+						'token.gcm': {
+							$exists: true,
+						},
+					},
+				],
+			},
+		],
+	};
+
+	const tokens = await AppsTokens.col.countDocuments(query);
```

Comment:
pls move the query to the model class.. the method name can be smth like `countTokensByUserId(userId: string)`

Reply:
Passing by to say that `BaseRaw` has a `countDocuments` method, so no need to use `.col.countDocuments`, just `Model.countDocuments`.

Reply:
Done! I used `countDocuments` from `BaseRaw`